source/index.html

```diff
@@ -1,12 +1,206 @@
+                <cite class="reviews__author">
+                  Николай Петров
+                </cite>
+                <p class="reviews__text">
+                  «Это приложение перевернуло мой мир и позволило по-новому взглянуть на привычные серые вещи! А еще я познакомился со своей будущей женой в комментариях к выложенной фотографии!»
+                </p>
+              </blockquote>
+            </li>
+            <li class="reviews__item reviews__item--current slider__item">
+              <blockquote class="reviews__blockquote">
+                <cite class="reviews__author">
+                  Николай Петров
+                </cite>
+                <p class="reviews__text">
+                  «Это приложение перевернуло мой мир и позволило по-новому взглянуть на привычные серые вещи! А еще я познакомился со своей будущей женой в комментариях к выложенной фотографии!»
+                </p>
+              </blockquote>
+            </li>
+            <li class="reviews__item slider__item">
+              <blockquote class="reviews__blockquote">
+                <cite class="reviews__author">
+                  Николай Петров
+                </cite>
+                <p class="reviews__text">
+                  «Это приложение перевернуло мой мир и позволило по-новому взглянуть на привычные серые вещи! А еще я познакомился со своей будущей женой в комментариях к выложенной фотографии!»
+                </p>
+              </blockquote>
+            </li>
+          </ul>
+          <div class="reviews__toggles slider__toggles">
+            <button class="slider__toggle slider__toggle--active" type="button" aria-label="Первый отзыв"></button>
+            <button class="slider__toggle" type="button" aria-label="Второй отзыв"></button>
+            <button class="slider__toggle" type="button" aria-label="Третий отзыв"></button>
+          </div>
+        </div>
+      </section>
+      <section class="prices">
+        <div class="prices__wrapper slider">
+            <h2 class="visually-hidden">Стоимость</h2>
+            <!-- Тут должна быть таблица, но я пока не поняла, как ее сделать -->
+            <p class="prices__toggles slider__toggles">
+              <button class="slider__toggle slider__toggle--active" type="button" aria-label="База"></button>
+              <button class="slider__toggle" type="button" aria-label="Стандарт"></button>
+              <button class="slider__toggle" type="button" aria-label="Анлим"></button>
+            </p>
+        </div>
+      </section>
+      <section class="contacts">
+        <h2 class="visually-hidden">Контактная информация</h2>
+        <div class="contacts__wrapper">
+          <р class="contacts__slogan">Остались вопросы?</p>
+          <p class="contacts__slogan">Свяжитесь с нами!</p>
+          <ul class="contacts__list">
+            <li class="contacts__item contacts__item-phone">
+              <p class="contacts__title">
+                Звоните:
+              </b>
```

Comment:
```suggestion
              </p>
```
видимо хотела сделать жирным через теги